common/analysis/lint_waiver.cc

```diff
@@ -376,6 +376,19 @@ static absl::Status WaiveCommandHandler(
           waiver->WaiveLineRange(rule, line_start - 1, line_end);
         }
 
+	if (!can_use_regex && !can_use_lineno) {
+	  unsigned int number_of_lines = 0;
```

Comment:
This initialization is dead, might as well declare and set value where it is first used a few lines below.
We can take care of that on importing this PR.